game-app/game-core/src/main/java/games/strategy/triplea/delegate/TechTracker.java

```diff
@@ -1,17 +1,68 @@
 package games.strategy.triplea.delegate;
 
 import games.strategy.engine.data.Change;
+import games.strategy.engine.data.GameData;
 import games.strategy.engine.data.GamePlayer;
 import games.strategy.engine.data.TechnologyFrontier;
+import games.strategy.engine.data.UnitType;
 import games.strategy.engine.data.changefactory.ChangeFactory;
 import games.strategy.engine.delegate.IDelegateBridge;
+import games.strategy.triplea.attachments.TechAbilityAttachment;
 import games.strategy.triplea.attachments.TechAttachment;
 import java.util.ArrayList;
 import java.util.Collection;
+import lombok.AllArgsConstructor;
 
 /** A collection of methods for tracking which players have which technology advances. */
-public final class TechTracker {
-  private TechTracker() {}
+@AllArgsConstructor
+public class TechTracker {
+  private final GameData data;
+
+  public int getAirDefenseBonus(GamePlayer player, UnitType type) {
+    return TechAbilityAttachment.getAirDefenseBonus(type, getCurrentTechAdvances(player));
+  }
+
+  public int getAirAttackBonus(GamePlayer player, UnitType type) {
+    return TechAbilityAttachment.getAirAttackBonus(type, getCurrentTechAdvances(player));
+  }
+
+  public int getMovementBonus(GamePlayer player, UnitType type) {
+    return TechAbilityAttachment.getMovementBonus(type, getCurrentTechAdvances(player));
+  }
+
+  public int getAttackBonus(GamePlayer player, UnitType type) {
+    return TechAbilityAttachment.getAttackBonus(type, getCurrentTechAdvances(player));
+  }
+
+  public int getAttackRollsBonus(GamePlayer player, UnitType type) {
+    return TechAbilityAttachment.getAttackRollsBonus(type, getCurrentTechAdvances(player));
```

Comment:
Those static methods from `TechAbilityAttachment` seem really odd. The look like they have nothing to do with the actual class except that they all (?) share the helper method `TechAbilityAttachment#sumIntegerMap`.
I believe the newly created `TechTracker` class might be suited better for a lot of them, i.e. inlining the static methods might make the code cleaner.

I have to admit though, I'm not sure how  tangled up some of the helper methods are so maybe the rabbit hole of refactoring is way too deep for this PR. So please tell me what you think about this idea and if you'd consider it worth addressing